server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -112,7 +112,22 @@ public enum MetadataVersion {
     //
 
     // Enables ELR by default for new clusters (KIP-966).
-    IBP_4_1_IV0(26, "4.1", "IV0", false);
+    IBP_4_1_IV0(26, "4.1", "IV0", false),
+
+    // Enables share groups. Note, share groups are for preview only in 4.1. (KIP-932).
+    IBP_4_1_IV1(27, "4.1", "IV1", false),
```

Comment:
@AndrewJSchofield : @jolshan and I were wondering why this particular MV was added. Does it depend on any records/RPCs controlled by MV?

Reply:
@AndrewJSchofield : Thanks for the reply. Before feature was properly introduced, all inter broker RPCs and record formats are gated by MVs. When we started to use features, the intention was to break up the RPCs and records by feature. For example, all RPCs used by the group coordinator and all records used in the offset topic are gated by GV, not MV. Similarly, all RPCs used by Qfk should be gated by SV, not MV. Could you submit a followup PR on this?

Reply:
+1 to Jun's comment

Reply:
Yes. I will do this shortly.